src/lib.rs

```diff
@@ -30,7 +37,11 @@ fn run_game_loop(config: &mut Config, guess: char, player_state: &mut PlayerStat
     return GameState::PLAYING;
 }
 
-fn update_correctly_guessed_letters(config: &Config, player_state: &mut PlayerState, guess: char) {
+fn update_correctly_guessed_letters(
+    config: &config::Config,
```

Comment:
Same here and in the `run` function below

Reply:
updated!